src/list/ListView.jsx

```diff
@@ -51,6 +58,32 @@ function ListView() {
       .finally(() => setLoading(false));
   }
 
+  function onChangeFilter(value) {
+    setFilterObject(Object.assign( {}, value ));
+  }
+
+  const debouncedFilterChangeHandler = useCallback(debounce(onChangeFilter, 400), []);
+
+  function getPagesWithQuery() {
+    let queryString = '';
+    if(filterObject.name) {
+        queryString += `name=${filterObject.name}`;
```

Comment:
suggestion: Adding a trailing `&` could make the rest of the logic in this method more simple (and shouldn't have a negative side effect)
```suggestion
        queryString += `name=${filterObject.name}&`;
```

Reply:
Suggestion taken.